src/pages/_Components/EnsonoStacksImage/EnsonoStacksImage.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Link from "@docusaurus/Link";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import AmidoStacksLogo from './AmidoStacksLogo.svg'
+
+import customStyle from './EnsonoStacksImage.module.css'
+
+const text = {
+    title: 'Ensono Stacks',
+    description: 'Making digital transformation easier and more reliable.',
+    buttonText: 'Show me how'
+}
+
+const EnsonoStacksImage = () => {
+    const baseUrl = useBaseUrl("docs/");
+
+    return (
+        <div className={customStyle.ensonoStacksImageContainer}>
+            <div className={customStyle.wrapperImage}>
+                <img src={useBaseUrl('img/ensonoImage.png')} alt="Ensono Image"/>
+            </div>
+            <div className={customStyle.wrapper}>
+                <AmidoStacksLogo/>
+                {/* <h1>{text.title}</h1> */}
+                <p>{text.description}</p>
+                <div style={{ marginTop: 20 }}>
```

Comment:
Would we want to move this style into a `css` file?

Reply:
done